release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1723,6 +1722,82 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously, if you configured user-defined outbound routing when installing a private Azure cluster, the cluster was incorrectly deployed with the default public load balancer. This behavior occurred when using the installer-provisioned infrastructure to install the cluster. With this update, the installation program no longer creates the public load balancer when user-defined routing is configured. (link:https://issues.redhat.com/browse/OCPBUGS-9404[*OCPBUGS-9404*])
+
+* Previously, the vSphere Terraform `vsphere_virtual_machine` resource did not include the `firmware` parameter. This issue caused the firmware of the virtual machine (VM) to be set to `bios` by default instead of `efi`. Now, the resource includes the `firmware` parameter and sets `efi` as the default value for the parameter, so that the VM runs the Extensible Firmware Interface (EFI) instead of the basic input/output system (BIOS) interface. (https://issues.redhat.com/browse/OCPBUGS-9378[OCPBUGS-9378*)
+
+* Previously, for clusters that run on {rh-openstack}, in the deprovisioning phase of installation, the installer deleted Object storage containers sequentially. This behavior caused slow and inefficient deletion of objects, especially with large containers. This problem occurred in part because image streams that use Swift containers accumulated objects over time. Now, bulk object deletion now occurs concurrently with up to 3 calls to the {rh-openstack} API, improving efficiency by handling a higher object count per call. This optimization speeds up resource cleanup during deprovisioning. (link:https://issues.redhat.com/browse/OCPBUGS-9081[*OCPBUGS-9081*])
+
+* Previously, the installation program did not exit with an error if you installed a cluster on Azure using disk encryption without providing a subscription ID. This caused the installation to begin only to fail later on. With this update, the installation program requires you to specify a subscription ID for encrypted Azure installations and exits with an error if you do not provide one. (link:https://issues.redhat.com/browse/OCPBUGS-8449[*OCPBUGS-8449*])
+
+* Previously, the Agent-based installer showed the results of secondary checks such as ping and nslookup, which can harmlessly fail even when the installation succeeds. This could result in errors being displayed despite the cluster installing successfully. With this update, secondary checks only display their results if the primary installation checks fail, so that you can use the secondary checks to troubleshoot the failed installation. (link:https://issues.redhat.com/browse/OCPBUGS-8390[*OCPBUGS-8390*])
```

Comment:
```suggestion
* Previously, the Agent-based installer showed the results of secondary checks such as `ping` and `nslookup`, which can harmlessly fail even when the installation succeeds. This could result in errors being displayed despite the cluster installing successfully. With this update, secondary checks only display results if the primary installation checks fail, so that you can use the secondary checks to troubleshoot the failed installation. (link:https://issues.redhat.com/browse/OCPBUGS-8390[*OCPBUGS-8390*])
```